### DISCUSSION TITLE
Provide alternative for npx script to sync values

### DISCUSSION THREAD

**Discussion by simonschaufi** [envault/envault#10] [Ideas]
**Is your feature request related to a problem? Please describe it**
I want to use the npx sync script on a shared webhosting server where nodejs is not installed.

**Describe the solution you'd like**
Have a php script/phar or any other kind of binary file that I can execute without node js dependency.

I bought this package in the hope that it is ready to be used but now I'm stuck as it has dependencies that are not clear before buying it.

**Comment by Jaxkr** [comment]
Could be wrong, but I don't think envault is intended for deploying to production.

**Comment by simonschaufi** [comment]
Even if it's not intended, it would be great to use it in production as i can add the sync command into my deployment pipeline and manage my .env within the application without always ssh-ing into the server and change the values there considering that .env values also get cached.

**Comment by danharrin** [comment]
Hey, we've decided that this feature does not fit with the direction that we would like to take the project. The API for syncing your environment is not guarded, so I've open sourced the [NPX CLI](https://github.com/envault/cli). If you wish, you can review the code there to find out how to make an API call to Envault server from your own script. Many thanks!
